pkg/version/manifest.go

```diff
@@ -0,0 +1,36 @@
+package version
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+const (
+	operatorHomePath = "OPERATOR_HOME"
+	manifestFile     = "MANIFEST"
+)
+
+var manifest = make(map[string]string)
+
+func init() {
```

Comment:
There's little guarantee for the `init()` method - especially with two such methods in the same module. Instead of reading in init(), can you just lazily read it? First time, `GetOperationVersion` is called, read the manifest and keep it locally.

Reply:
That would also let you return error in case the build was messed up vs panic.